gradle/scripts/dependencies.gradle

```diff
@@ -120,20 +49,20 @@ dependencies {
     compile "li.cil.oc:OpenComputers:MC${minecraft_version}-${opencomputers_version}:api"
     compile "net.industrial-craft:industrialcraft-2:${ic2_version}-experimental:api"
     compile "net.mcft.copy.betterstorage:BetterStorage:${minecraft_version}-${betterstorage_version}:api"
+	compile "coloredlightscore:ColoredLightsCore:1.3.7.39:api"
```

Comment:
Strange. Colored Lights was the first thing I changed, because they now have their own repo.
I faintly remember putting the version into the `gradle.properties`, but that might just while trying it...